std_msgs/Header header
uint32 itow
uint16 week_num
int32 lat
int32 lon
int32 hgt
float32 vn
float32 ve
float32 vd
float32 roll
float32 pitch
float32 yaw
float32 rtk_yaw
float32 wheel_angle
float32[7] imu
uint8 fix_type
uint8 sv_num
uint8 diff_age
uint8 heading_type
uint16 pos_acc
uint16 status
